src/providers/update-local-school-data/update-local-school-data.ts

```diff
@@ -223,6 +223,27 @@ export class UpdateLocalSchoolDataProvider {
     }
   }
 
+  async updateSubmissionIdArr(submissionId) {
+    await this.localStorage
+      .getLocalStorage(storageKeys.submissionIdArray)
+      .then(async (arr) => {
+        Array.isArray(submissionId)
```

Comment:
Why is this check?

Reply:
when migrating then i will pass the array of submissionId other wise string value will come